### PR TITLE
Adding many new weapons

### PR DESCRIPTION
**Pull request by AirwavesMan**
Adding these weapons to epoch with ammo:

Mk16 Aimpoint SD
Mk16 CQC
Mk16 CQC EOTech
Mk16 EGLM EOTech,(40mm)
Mk16 EGLM RCO,(40mm)
Mk16 EOTech
Mk16 Mk4CQ/T
Mk16 EGLM EOTech
Mk17 Sniper
Mk17 Aimpoint
Mk17 Aimpoint SD
Mk17 EGLM ACOG,(40mm)
L85A2 ACOG
L86A2 ACOG LMG
AA12 Military Shotgun
M110_NVG_EP1

### CODE REVIEW

**Comment by ebaydayz** [comment]
The only thing is the AA12 HE rounds may be a bit OP. Maybe someone else can comment about that. I have not played with it enough to know.

**Comment by icomrade** [comment]
The AA12 is a great gun but I don't think HE rounds should be in by default, just slugs and buckshot. HE rounds basically makes it a flak cannon

**Comment by AirwavesMan** [comment]
Ive played with it a bit. So you are right it is a way too OP, I will comment it out for loot and trader. Btw I added the UH60M_MEV_EP1 to the trader. Github added it to that request.

File: SQF/dayz_code/Configs/CfgLoot/Groups/Weapons.hpp [hunk 1]
```diff
@@ -55,7 +55,10 @@ sniperrifles[] = {
 	{Loot_WEAPON,		0.5,	VSS_vintorez},
 	{Loot_WEAPON,		0.6,	SCAR_H_LNG_Sniper_SD},
 	{Loot_WEAPON,		1,		M14_DZ}, //M14_EP1
-	{Loot_WEAPON,		1,		DMR_DZ} //DMR
+	{Loot_WEAPON,		1,		DMR_DZ}, //DMR
```
Comment (ebaydayz) [inline SQF/dayz_code/Configs/CfgLoot/Groups/Weapons.hpp]: DMR rate should probably be the same or lower than the M24 and M40.

**Comment by ebaydayz** [comment]
We are going to need to add the new UH60 to the configs with the other UH60s to disable the radar and all that good stuff.

**Pull request merged**